cypress/integration/preserve-auth0-cookies-once.spec.js

```diff
@@ -1,30 +1,50 @@
-/**
- * Cypress automatically clears all cookies before each test to prevent state
- * from building up, that's why we test preserving cookies here.
- */
-context('Cookie', () => {
-  before(() => {
-    cy.login();
-  });
+const tests = [
+  {
+    title: 'Cookie',
+    credentials: undefined,
+  },
+  {
+    title: 'Split Cookie',
+    credentials: {
+      username: Cypress.env('auth0SplitCookieUsername'),
+      password: Cypress.env('auth0SplitCookiePassword'),
+    },
+  },
+];
 
-  beforeEach(() => {
-    cy.preserveAuth0CookiesOnce();
-  });
+for (const { title, credentials } of tests) {
```

Comment:
Please, use Jests `it.each(table)(name, fn)` ([docs](https://jestjs.io/docs/api#testeachtablename-fn-timeout)) syntax instead here.

Reply:
No worries, I should've probably linked to the [`describe.each` docs](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) and not `it.each`, sorry about that.

Here's a basic example:

```
context.each([
  ['Cookie', { credentials: undefined }],
  [
    'Split Cookie',
    {
      credentials: {
        username: Cypress.env('auth0SplitCookieUsername'),
        password: Cypress.env('auth0SplitCookiePassword'),
      },
    },
  ],
])('%s', (_, credentials) => {

/* test code */
})
```

Reply:
🤦  What was I thinking, of course, they are not using Jest, you are right. Sorry for the confusion. I had to deal with lots of tests written in tests lately. I must have got something mixed up. 😅

Reply:
No worries! 😄